### PR TITLE
dlv: Misc refactors

### PR DESCRIPTION
**Pull request by krhubert**

### CODE REVIEW

File: terminal/terminal.go [hunk 1]
```diff
@@ -158,10 +154,10 @@ func (t *Term) promptForInput() (string, error) {
 func (t *Term) handleExit() (int, error) {
 	fullHistoryFile, err := config.GetConfigFilePath(historyFile)
 	if err != nil {
-		fmt.Println("Error saving history file:", err)
+		fmt.Println("Error saving history file: ", err)
```
Comment (derekparker) [inline terminal/terminal.go]: `fmt.Println` automatically inserts a space separator in between arguments.

**Comment by derekparker** [comment]
Sorry for the delay, I've been swamped lately. I scanned this and it seems fine, will take a closer look this evening or tomorrow once I'm caught up on a few things. Saw one minor thing which I pointed out, but as I said I mostly just scanned this.

File: terminal/terminal.go (hunk 1 above)
Comment (krhubert) [inline terminal/terminal.go]: True, so 5 lines belowe is fmt.Println("readline history error: ", err), which has space, I will fix this

**Comment by derekparker** [comment]
Merged, thanks.

**Comment by derekparker** [comment]
@krhubert I also went ahead and took care of the `Println` formatting.

**Pull request closed without merging**